bundles/org.eclipse.ui.navigator/META-INF/MANIFEST.MF

```diff
@@ -18,7 +18,8 @@ Export-Package: org.eclipse.ui.internal.navigator;x-friends:="org.eclipse.ui.nav
 Require-Bundle: org.eclipse.core.runtime;bundle-version="[3.18.0,4.0.0)",
  org.eclipse.ui;bundle-version="[3.6.0,4.0.0)",
  org.eclipse.core.expressions;bundle-version="[3.4.0,4.0.0)",
- org.eclipse.jface;bundle-version="[3.18.0,4.0.0)"
+ org.eclipse.jface;bundle-version="[3.18.0,4.0.0)",
+ org.eclipse.core.resources;bundle-version="3.18.0"
```

Comment:
This is not desired. The navigator may be used in contexts without resources.

Reply:
Ok, latest patch removes this requirement.